lib/openapi3.js

```diff
@@ -179,6 +179,9 @@ function getParameters(data) {
     if (!Array.isArray(data.parameters)) data.parameters = [];
     data.longDescs = false;
     for (let param of data.parameters) {
+        if (typeof param !== 'object') {
+            continue;
```

Comment:
Maybe we could do something like this?:
```suggestion
            param = {};
```